packages/application-components/src/components/public-page-layout/public-page-layout.spec.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { renderComponent, screen } from '../../test-utils';
+import PublicPageLayout from './public-page-layout';
+
+describe('rendering', () => {
+  it('should render welcome message and footer', async () => {
+    renderComponent(
+      <PublicPageLayout welcomeMessage="Welcome to the Merchant Center">
+        {'Test'}
```

Comment:
Nit: should this work?

```suggestion
        Test
```